badge.css

```diff
@@ -0,0 +1,164 @@
+
+body{
+    position: relative;
+    margin-left: 4px;
+    width: 100%;
+}
+.container{
+    /* background-color:skyblue ; */
+    left: 0;
+    padding: 3px;
+    display: flex;
+    gap: 5px;
+    position: relative;
+    /* z-index: 1; */
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    margin: 5px;
```

Comment:
use consistent unit somewhere u are using px somewhere rem
i will suggest to stick with rem